investors/views.py

```diff
@@ -215,6 +218,156 @@ def perform_destroy(self, instance):
         )
         super().perform_destroy(instance)
 
+
+class FollowedProjectViewSet(viewsets.ModelViewSet):
+    """
+    ViewSet for managing FollowedProject instances.
+    Only authenticated investors who own the FollowedProject can modify/delete it.
+    """
+    permission_classes = [IsAuthenticatedInvestor403]
+    authentication_classes = [CookieJWTAuthentication]
+    serializer_class = FollowedProjectSerializer
+    http_method_names = ['get', 'post', 'put', 'patch', 'delete', 'head', 'options']
+
+    def get_queryset(self):
+        user = self.request.user
+        if hasattr(user, 'investor') and user.investor:
+            return FollowedProject.objects.filter(investor=user.investor)
+        return FollowedProject.objects.none()
+
+    def create(self, request, *args, **kwargs):
+        user = request.user
+        payload = request.data
+
+        if "project" not in payload or payload.get("project") in (None, "", []):
+            logger.warning(
+                "FollowedProject create failed: missing project",
+                extra={"by_user": user.pk},
+            )
+            return Response(
+                {"project": ["This field is required."]}, 
+                status=status.HTTP_400_BAD_REQUEST
+            )
+
+        project_id = payload.get("project")
+        if project_id and FollowedProject.objects.filter(
+                investor=user.investor, project_id=project_id
+        ).exists():
+            logger.warning(
+                "FollowedProject create failed: duplicate",
+                extra={"investor_id": user.investor.pk, "project_id": project_id, "by_user": user.pk},
+            )
+            existing = FollowedProject.objects.get(investor=user.investor, project_id=project_id)
+            return Response(
+                FollowedProjectSerializer(existing).data,
+                status=status.HTTP_200_OK
+            )
```

Comment:
The view checks for duplicates with a DB query manually before serializer validation, which is redundant with unique constraints. Rely on serializer validation and handle IntegrityError for clean logic.
```
try:
    serializer.is_valid(raise_exception=True)
    self.perform_create(serializer)
except IntegrityError:
    # handle duplicate gracefully
```